src/group/libp2p_group_gossip_handler.erl

```diff
@@ -0,0 +1,10 @@
+-module(libp2p_group_gossip_handler).
+
+
+-callback init_gossip_data(State::any()) -> init_result().
+-callback handle_gossip_data(Msg::binary(), State::any()) -> handler_result().
+
+-type init_result() :: ok | {send, binary()}.
+-type handler_result() :: ok | {send, binary()} | {error, term()}.
```

Comment:
If I do a `{send, binary()}` return here, where does it go?

Reply:
heh, right now, nowhere. The idea I had at the time was that the handler fot eh given gossip data could return some more data to send to the gossip group? Is that useful?